### DISCUSSION TITLE
Become a Pokemon Master!

### DISCUSSION THREAD

**Discussion by manzoku23** [NAIST-SE/AttractingContributors#69] [General]
Please write down the Pokémon you know.
English name or japanese name is OK.
Please feel free to make a pull request!!

https://github.com/Exe-dev/AttractingContributors-POKEMON

**Comment by sergiodl-naist** [comment]
Added my all time favorite!
Please visit and try to ask a question or recommend one vtuber on my fork!
https://github.com/sergiodl-naist/AttractingContributors-VTubers

**Reply by manzoku23** [reply]
thx :D
Add my favorite Vtuber!

**Reply by sergiodl-naist** [reply]
Thank you!

**Comment by yokoro13** [comment]
I added!
Please help me!
https://github.com/yokoro13/AttractingContributors-VideoGames

**Reply by manzoku23** [reply]
thx :D
I added my favorite one!

**Comment by iikyara** [comment]
I added!
Please help me too!
https://github.com/iikyara/AttractingContributors-Gifts

**Reply by manzoku23** [reply]
thx  :D
I added my favorite one!
